modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,380 @@
+      # a failure.  This annotation works in tandem with the toleration below.
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+      labels:
+        app: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+    spec:
+      serviceAccount: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+      serviceAccountName: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+      containers:
+      - image: image-registry.openshift-image-registry.svc:5000/{{.Values.specialresource.spec.namespace}}/{{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}:v{{.Values.kernelFullVersion}}
+        name: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+        imagePullPolicy: Always
+        command: ["/sbin/init"]
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@{{.Values.specialresource.metadata.name}}"]
+        securityContext:
+          privileged: true
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+        feature.node.kubernetes.io/kernel-version.full: "{{.Values.KernelFullVersion}}"
+----
+
+.. Change into the `chart/simple-kmod-0.0.1` directory
++
+[source, terminal]
+----
+cd ..
+----
+
+.. Save the following YAML for the Chart as `Chart.yaml` in the `chart/simple-kmod-0.0.1` directory.
++
+[source, yaml]
+----
+apiVersion: v2
+name: simple-kmod
+description: Simple kmod will deploy a simple kmod driver-container
+icon: https://avatars.githubusercontent.com/u/55542927
+type: application
+version: 0.0.1
+appVersion: 1.0.0
+----
+
+. Now that the structure for the Helm chart is prepared, create the chart with the helm package command, from the `chart` directory:
++
+[source,terminal]
+----
+cd ..
+helm package simple-kmod-0.0.1/
+----
++
+.Example output
+[source,terminal]
+----
+Successfully packaged chart and saved it to: /data/dagray/git/dagrayvid/special-resource-operator/yaml-for-docs/chart/simple-kmod-0.0.1/simple-kmod-0.0.1.tgz
+----
+
+. Create a ConfigMap to store the chart files
++
+[source,terminal]
+----
+mkdir cm
+cp simple-kmod-0.0.1.tgz cm/simple-kmod-0.0.1.tgz
+helm repo index cm --url=cm://simple-kmod/simple-kmod-chart
+oc create namespace simple-kmod
+oc create cm simple-kmod-chart --from-file=cm/index.yaml --from-file=cm/simple-kmod-0.0.1.tgz -n simple-kmod
+----
+
+
+. To deploy the simple-kmod using the Helm chart we created in the ConfigMap, the following SpecialResource manifest can be used. The `spec.chart.repository.url` field tells SRO to look for the chart in a ConfigMap. Optionally, uncomment the #debug: true line, to have the YAML files in the chart printed in full in the operator logs, to verify they are being created and templated properly. Save this YAML as `simple-kmod-configmap.yaml`
```

Comment:
1) Reword 1st sentence slightly to change verb tense:

Use the following SpecialResource manifest to deploy the simple-kmod using the Helm chart that you created in the ConfigMap.

2)
s/operator logs, to verify they are being created and templated properly./operator logs and to verify that the logs are created and templated properly./